contracts/schemes/Join.sol

```diff
@@ -174,11 +150,11 @@ contract JoinAndQuit is
     returns(bytes32)
     {
         address proposer = msg.sender;
-        require(fundings[proposer].state != MemeberState.Candidate, "already a candidate");
-        require(fundings[proposer].state != MemeberState.Accepted, "accepted and not redeemed yet");
+        require(membersState[proposer] != MemeberState.Candidate, "already a candidate");
```

Comment:
maybe add a subject to the error message so we know what it is about? ("proposer is already a candidate")